code/src/main/kotlin/lexer/Lexer.kt

```diff
@@ -7,7 +7,7 @@ import kotlin.coroutines.experimental.buildSequence
  */
 class Lexer(private val inputContent: String) : ILexer {
     private val endLines = listOf("\n","\r\n",";")
-    private val specialChars = listOf("=", "[", "]", "(", ")", "{", "}", ",", "\\", "->", "\\") + endLines
+    private val specialChars = listOf("=", "[", "]", "(", ")", "{", "}", ",", "\\", ":", "\\") + endLines
```

Comment:
Nice!

Reply:
this has been fixed multiple places :P prolly gon get merge conflicts but ohwell